README.md

```diff
@@ -58,27 +58,34 @@ See [Release procedure](CONTRIBUTING.md#release-procedure) for more information
 
 ## Join an active Network
 
-To get the latest `palomad` binary:
+> #### Note:
+> Some have seen errors with GLIBC version differences with the downloaded binaries for v1.0.0.  This is caused by a difference in the libraries of the host that built the binary and the host running the binary.
+>
+> We did not find this bug in time to fix it in the v1.0.0 release.  Instead there is a workaround.  If you experience these errors, please pull down the code and build it, rather than downloading the prebuilt binary
+
+### To get the latest prebuilt `palomad` binary:
 
-Testnet:
 ```shell
-wget -O - https://github.com/palomachain/paloma/releases/download/v0.11.7/paloma_Linux_x86_64.tar.gz  | \
-  sudo tar -C /usr/local/bin -xvzf - palomad
+wget -O - https://github.com/palomachain/paloma/releases/download/v1.0.0/paloma_Linux_x86_64.tar.gz  | \
+  sudo tar -C ~/ -xvzf - palomad
```

Comment:
This doesn't extract to /usr/local/bin anymore, but to user home directory. Any reason why we are making this change?